resolvedTasks/task5.php

```diff
@@ -0,0 +1,28 @@
+<?php 
+/*Arrays Part 2
+
+to create an array that's empty, you can use the following approaches
+
+Just BRACKETS: (php version > 5.4)
+$arrayNameVariable = [];
+A FUNCTION. In orevious versions of php (in php version <= 5.4) you can use the array() function like so:
+$arrayNameVariable = array();
+EXAPLES
+
+with brackets
+$arrA = ["first value","second","abc","Yes please!!!"];
+As a function $arrA = array("first value","second","abc","Yes please!!!");
+
+Both examples do the exact same thing: a array variable with the 3 options.
+
+TASK:
+
+Create 1 array with values from 3 to 10, in both ways; You can do this manually, no need for a loop or a function use.
+*/
+$arrA = [3,4,5,6,7,8,9,10];
+foreach ($arrA as $key => $value) {
```

Comment:
I said no loops necessary
Why did you add a foreach loop?

- [ ] REMOVE the loop